.github/workflows/windows.yml

```diff
@@ -0,0 +1,36 @@
+name: github-windows
+
+on:
+  push:
+  pull_request:
+
+concurrency:
+  group: ${ {github.event_name }}-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{github.event_name == 'pull_request'}}
+
+permissions:
+  contents: read
+
+jobs:
+  windows-cuda:
+    # Cuda build on Windows
+    name: Windows Cuda
```

Comment:
can we make the name consistent with the names of the other CI pipelines?

Reply:
so that is looks consistent with the other names 

![Uploading image.png…]()

Reply:
What do you want it to be? `github-windows-cuda / CI`?